docs/core/install/includes/package-manager-failed-to-fetch-deb.md

```diff
@@ -0,0 +1,2 @@
+
+While installing the .NET Core package, you may see an error similar to `Failed to fetch ... File has unexpected size ... Mirror sync in progress?`. Generally speaking, this error means that the package feed for .NET Core is being upgraded with newer package versions, and that you should try again later. During an upgrade, the package feed should not be unavailable for more than 30 minutes. If you continually receive this error for more than 30 minutes, please file an issue at <https://github.com/dotnet/core/issues>.
```

Comment:
We usually avoid use of "please".

Reply:
@tdykstra  I know. Acro complains that it doesn't add anything. But in this case, I think it does. We're really requesting users to file an issue so that we can be alerted to the problem. Thoughts?

Reply:
It's your call -- if you think it's justified in this instance I'm fine with it. I agree it would sound a bit abrupt here without it.

Reply:
I agree it makes sense for this message. Style guide on [please](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please):

> Avoid please except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation.
> 
> Example
> The network connection was lost. Please reenter your password.

The service is to blame, and it's inconvenient to have to file an issue to let us know.